apis/plugins/ContainerPlugin.go

```diff
@@ -1,12 +1,16 @@
 package plugins
 
-import "io"
+import (
+	"io"
+
+	"github.com/alibaba/pouch/apis/types"
+)
 
 // ContainerPlugin defines places where a plugin will be triggered in container lifecycle
 type ContainerPlugin interface {
 	// PreCreate defines plugin point where receives a container create request, in this plugin point user
 	// could change the container create body passed-in by http request body
-	PreCreate(io.ReadCloser) (io.ReadCloser, error)
+	PreCreate(*types.ContainerCreateConfig) error
```

Comment:
Interface signature and return value are all different from origin, some files missing?

Reply:
@xiaoxubeii You are right...